README.md

```diff
@@ -1,12 +1,14 @@
-[![Vulnerabilities](https://sonarcloud.io/api/project_badges/measure?project=epiphany-platform_epiphany&metric=vulnerabilities)](https://sonarcloud.io/summary/new_code?id=epiphany-platform_epiphany)
-[![Code Smells](https://sonarcloud.io/api/project_badges/measure?project=epiphany-platform_epiphany&metric=code_smells)](https://sonarcloud.io/summary/new_code?id=epiphany-platform_epiphany)
+[![GitHub release](https://img.shields.io/github/v/release/hitachienergy/epiphany.svg)](https://github.com/hitachienergy/epiphany/releases)
+[![Github license](https://img.shields.io/github/license/hitachienergy/epiphany)](https://github.com/hitachienergy/epiphany/releases)
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=hitachienergy_epiphany&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=hitachienergy_epiphany)
+[![Maintainability Rating](https://sonarcloud.io/api/project_badges/measure?project=hitachienergy_epiphany&metric=sqale_rating)](https://sonarcloud.io/summary/new_code?id=hitachienergy_epiphany)
+[![Security Rating](https://sonarcloud.io/api/project_badges/measure?project=hitachienergy_epiphany&metric=security_rating)](https://sonarcloud.io/summary/new_code?id=hitachienergy_epiphany)
+[![Bugs](https://sonarcloud.io/api/project_badges/measure?project=hitachienergy_epiphany&metric=bugs)](https://sonarcloud.io/summary/new_code?id=hitachienergy_epiphany)
+[![Vulnerabilities](https://sonarcloud.io/api/project_badges/measure?project=hitachienergy_epiphany&metric=vulnerabilities)](https://sonarcloud.io/summary/new_code?id=hitachienergy_epiphany)
+[![Code Smells](https://sonarcloud.io/api/project_badges/measure?project=hitachienergy_epiphany&metric=code_smells)](https://sonarcloud.io/summary/new_code?id=hitachienergy_epiphany)
+
+**⚠️ Epiphany is no longer under active development, no new features or upgrades will be done by the core team. The core team however, will address critical defects and security issues during the [LTS versions](docs/home/LIFECYCLE.md) lifecycle. Finally the core team will continue the [governance](docs/home/GOVERNANCE.md) of external contributions and publish intermittent releases for [LTS versions](docs/home/LIFECYCLE.md) during their lifecycle.⚠️**
 
 **⚠️ Epiphany is no longer under active development, no new features or upgrades will be done by the [core team](https://github.com/orgs/epiphany-platform/teams/epiphany-team). The [core team](https://github.com/orgs/epiphany-platform/teams/epiphany-team) however, will address critical defects and security issues during the [LTS versions](docs/home/LIFECYCLE.md) lifecycle. Finally the  [core team](https://github.com/orgs/epiphany-platform/teams/epiphany-team) will continue the [governance](docs/home/GOVERNANCE.md) of external contributions and publish intermittent releases for [LTS versions](docs/home/LIFECYCLE.md) during their lifecycle.⚠️**
```

Comment:
Paragraph has been duplicated rather than new version replacing the old

Reply:
Duplicate introduced during rebasing. Fixed.